src/acom_music_box/conditions.py

```diff
@@ -117,9 +108,7 @@ def from_UI_JSON(self, UI_JSON, species_list, reaction_list):
     def from_config_JSON(
             self,
             path_to_json,
-            config_JSON,
-            species_list,
-            reaction_list):
+            object):
```

Comment:
Could you update the descriptions for the arguments? `line 121-122` is no longer needed and we can add a description for `object`.

Reply:
Yes